app/helpers/session_timeout_warning_helper.rb

```diff
@@ -13,7 +13,7 @@ def warning
 
   def timeout_refresh_url
```

Comment:
Rename this to `timeout_refresh_path`?

Reply:
Good idea! Updated in 4ee97bbb788365805396818da4c3e3058072450a